src/views/publish/upload-lookup.ejs

```diff
@@ -0,0 +1,62 @@
+<%- include("../partials/header", t); %>
+
+
+<main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-width-container app-width-container">
+        <div class="top-links">
+            <div class="govuk-width-container">
+                <a href="<%= buildUrl(`/publish/${locals.datasetId}/dimension-data-chooser/${dimension.id}`, i18n.language) %>" class="govuk-back-link"><%= t('buttons.back') %></a>
+                <a href="<%= buildUrl(`/publish/${locals.datasetId}/tasklist`, i18n.language) %>" class="govuk-link return-link"><%= t('publish.header.overview') %></a>
+            </div>
+        </div>
+
+        <p class="govuk-heading-s region-subhead"><%= locals.dimension.metadata.name %></p>
+        <h1 class="govuk-heading-xl"><%= t('publish.upload.lookup_heading') %></h1>
+
+        <%- include("../partials/error-handler"); %>
+
+        <% if (locals?.data) { %>
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full with-overflow">
+                    <%- include("../partials/dimension-preview-table.ejs", { dimension: locals.dimension, headers: locals.headers, data: locals.data }) %>
+                    <% if (locals.page_info?.total_records > locals.page_size) { %>
+                        <p class="govuk-body govuk-hint"><%= t('publish.measure_review.showing', {rows: locals.page_size, total: locals.page_info.total_records}) %></p>
+                    <% } %>
+                </div>
+            </div>
+
+            <div class="govuk-grid-row">
+                <div class="govuk-grid-column-full">
+                    <form method="post" role="continue" enctype="multipart/form-data">
+                        <fieldset class="govuk-fieldset">
+                            <div class="govuk-form-group">
+                                <label class="govuk-label govuk-label--m" for="csv">
+                                    <%= t('publish.upload.form.file.label') %>
+                                </label>
+                                <input class="govuk-file-upload" id="csv" name="csv" type="file" placeholder="Upload Data Files!" accept=".csv,.parquet,.json,.xls,.xlsx,.gz,text/csv,application/vnd.apache.parquet,application/parquet,application/json,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,application/msexcel,application/gzip">
```

Comment:
Accept should probably be a var that comes from a central place of allowed formats.

Reply:
Raise a ticket and we can move it out later... This isn't "new" code as such... Its just borrowed from the upload view.